www/_posts/2020-06-29-cordova-android-9.0.0.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova Android 9.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova Android 9.0.0`!  This is one of Cordova's supported platforms for building Android mobile applications.
+
+* [cordova-android@9.0.0](https://www.npmjs.com/package/cordova-android)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove android
+cordova platform add android@9.0.0
+```
+
+## Release Highlights
+
+* **Added Kotlin Support**
+
+Kotlin is one of the newest statically typed languages for developing Android apps. It is designed to work fully with the existing language, Java.
+
+By default, Cordova has Kotlin disabled but can be enabled with the `GradlePluginKotlinEnabled` `preference` tag.
```

Comment:
"but can be enabled" => "but it can be enabled"

"`preference` tag" => "preference in config.xml" (with link to preferences docs)